lib/timeline.js

```diff
@@ -0,0 +1,69 @@
+const guid = require('./utils/guid')
+
+class Timeline {
+  constructor(client) {
+    this.client = client
+  }
+
+  createEventType(applicationId, userId, data, cb) {
+    data['applicationId'] = data['applicationId'] || applicationId
```

Comment:
It seems a bit strange for the function to accept an `applicationId`and to check the data for an applicationId. Is it reasonable to remove this line `data['applicationId'] = data['applicationId'] || applicationId` and rely on the `applicationId` parameter?

Reply:
Potentially. My hesitation is that if you look at the hubspot api documentation it mentions that `applicationId` is a required property on the data posted to them. Rather then overriding this if a user has set this value per HubSpots api docs, I'm taking a more defensive approach of only setting this if the user hasn't.